### PR TITLE
Fixed FAQ section issue#3079

### LINKED ISSUE #3079: [FAQ] Correcting the FAQ Section
**Current Behavior**

None answers are correct and all the three are rather same.

image

**Desired Situation**

Correct answers are needed to be filled in to Improve User Experience.
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sourav-sharma7**
Signed-off-by: sourav-sharma7 <souravsharma1321@gmail.com>

**Description**
FAQ section fixed and button links added

This PR fixes #3079

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 703adda4a4a58e9cf3a480e1696bf351ef783866 at: https://6302fa4dab4f3a19abe375f4--layer5.netlify.app

File: src/assets/data/faq/index.js [hunk 1]
```diff
@@ -325,24 +325,22 @@ const data = {
...
       question: "How can I request a new integration?",
       category: "Meshery Integrations",
+      link: 'https://layer5.io/company/contact',
+      linktext: 'Contact Us',
       answer: [
-        "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
-      ],
-    },
-    {
-      question: "Does Meshery manage service meshes?",
```
Comment (Nikhil-Ladha) [inline src/assets/data/faq/index.js:342]: This question should be there as well.
The answer for the same is already provided by Debopriya in the issue comments, update accordingly.

File: src/assets/data/faq/index.js [hunk 2]
```diff
@@ -325,24 +325,22 @@ const data = {
...
     {
-      question: "How to Meshery Extension Points work?",
+      question: "How do Meshery Extension Points work?",
       category: "Meshery Integrations",
+      link: 'https://docs.meshery.io/extensibility',
+      linktext: 'Meshery Extensibility',
       answer: [
-        "Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.",
+        "Meshery is an extensible platform. Customize Meshery using any of its purpose-built extension points.",
+        'Take advantage of these extension points for working with different service meshes via adapters, different load generators and different providers. Use Meshery’s REST API or its GraphQL API both of which allow you to connect to any Kubernetes-native app.'
```
Comment (Nikhil-Ladha) [inline src/assets/data/faq/index.js:334]: Use double quotes

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/assets/data/faq/index.js (hunk 2 above)
Comment (sourav-sharma7) [reply src/assets/data/faq/index.js]: Done. Please have a look.

**Comment by l5io** [comment]
🚀 Preview for commit 97af9ecd3cee38cb0130e659f980a10ec33a9a37 at: https://63049625a5f44e4b4c625975--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**